portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -94,8 +99,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     Gson gson = new Gson();
+    String json = gson.toJson(comments);
+    json = gson.toJson(comments);
```

Comment:
Why do you need to convert comments to json twice?

Reply:
This was all done as a part of testing -- will remove them now and re-commit. Thank you!!